cf_deployment_functions.sh

```diff
@@ -83,4 +86,30 @@ unmap_blue_route() {
 set_app_version_environment_variable() {
   echo "# setting APP_VERSION=${APP_VERSION} for ${APP_FULL_NAME}"
   cf set-env ${APP_FULL_NAME} APP_VERSION "${APP_VERSION}"
-}
\ No newline at end of file
+}
+
+add_network_polices() {
+  suffix=$1
+  cat ${BIN_DIR}/network-policies.properties | grep ${APP_NAME} | while read route
+  do
+    # src and dest must have <<suffix>> replaced according to whether they are the application currently being deployed
+    src=$(echo ${route} | cut -d= -f1)
+    if [ ${src} = "${APP_NAME}<<suffix>>" ]; then
+      src=$(echo ${src} | sed "s/<<suffix>>/$suffix/g")
+    else
+      src=$(echo ${src} | sed "s/<<suffix>>/-${CF_SPACE}/g")
+    fi
+
+    dest=$(echo ${route} | cut -d= -f2)
+    if [ ${dest} = "${APP_NAME}<<suffix>>" ]; then
+      dest=$(echo ${dest} | sed "s/<<suffix>>/$suffix/g")
+    else
+      dest=$(echo ${dest} | sed "s/<<suffix>>/-${CF_SPACE}/g")
```

Comment:
"-${CF_SPACE}" should instead be "${SPACE_SUFFIX}" to work in production too

Reply:
Updated